### PR TITLE
add deps and more print out

### PR DESCRIPTION
**Pull request by robpre**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -54,9 +54,19 @@ def run(self):
...
 
-        
-        print(f"About to add timesheets between {self.date_range[0].isoformat()} and {self.date_range[1].isoformat()}")
-        input("Press ENTER to continue, CTRL+C to cancel")
+        projectRes = self.get(f"projects/{self.project_id}")
+        pName = projectRes.json()["name"]
+        activityRes = self.get(f"activities/{self.activity_id}")
+        aName = activityRes.json()["name"]
+
+        print(f"About to add 4h a weekday between {self.date_range[0].isoformat()} and {self.date_range[1].isoformat()} inclusive")
```
Comment (jthistle) [inline main.py]: I think the amount of time per day is configurable, no? Would prefer:

```py3
print(f"About to add {self.hours}h per weekday between {self.date_range[0].isoformat()} and {self.date_range[1].isoformat()} inclusive")
```

File: main.py [hunk 2]
```diff
@@ -66,11 +76,17 @@ def run(self):
...
-            projects_list += f"{project['id']}: {project['parentTitle']} - {project['name']}\n"
-        print(projects_list)
+            pt = project["parentTitle"]
+            groups[pt] = groups.get(pt, [])
+            groups[pt].append(f"\t{project['id']}: {project['parentTitle']} - {project['name']}")
+
+            # projects_list += f"\t{project['id']}: {project['parentTitle']} - {project['name']}\n"
+        for _, projects in groups.items():
+            for p in projects:
+                print(p)
```
Comment (jthistle) [inline main.py:89]: nice

**Review by jthistle — COMMENTED** [review]
cool stuff, just the one change to be made

File: main.py (hunk 1 above)
Comment (robpre) [reply main.py]:
```suggestion
        print(f"About to add {self.hours}h per weekday between {self.date_range[0].isoformat()} and {self.date_range[1].isoformat()} inclusive")
```

File: main.py [hunk 3]
```diff
@@ -66,11 +76,17 @@ def run(self):
...
         content = sorted(res.json(), key=lambda x: int(x["id"]))
+        groups = {}
 
-        projects_list = ""
         for project in content:
-            projects_list += f"{project['id']}: {project['parentTitle']} - {project['name']}\n"
-        print(projects_list)
+            pt = project["parentTitle"]
+            groups[pt] = groups.get(pt, [])
+            groups[pt].append(f"\t{project['id']}: {project['parentTitle']} - {project['name']}")
```
Comment (robpre) [inline main.py]:
```suggestion
            groups[pt].append(f"\t{project['id']}: {pt} - {project['name']}")
```

**Pull request merged**